steps/referandmonitor/login.ts

```diff
@@ -0,0 +1,22 @@
+import { Page, expect } from '@playwright/test'
+
+export const login = async (page: Page) => {
+    await page.goto(process.env.REFERANDMONITOR_URL)
+    await expect(page).toHaveTitle(/.*HMPPS Interventions - Referral - Referrals/)
+}
+
+export const loginAsSupplier = async (page: Page) => {
+    await page.goto(process.env.REFERANDMONITOR_URL)
+    await expect(page).toHaveTitle(/HMPPS Digital Services - Sign in/)
+    await page.fill('#username', process.env.REFERANDMONITOR_SUPPLIER_USERNAME!)
```

Comment:
Should we add sample values for these to the `.env` example in the README?  

Also, do we have values we can add to the CI pipeline (here: https://github.com/ministryofjustice/hmpps-probation-integration-services/blob/main/.github/workflows/pipeline.yml#L79-L92)?  I know we're not running them in CI yet, but might be good to get everything we need in now to make it easier when we do.

Reply:
Updated the docs and will do a PR for the pipeline